test/node/features/express.feature

```diff
@@ -33,6 +33,9 @@ Scenario: an asynchronous thrown error in a route
   And the exception "message" equals "async"
   And the exception "type" equals "nodejs"
   And the "file" of stack frame 0 equals "scenarios/app.js"
+  And the event "request.url" equals "http://express/async"
+  And the event "request.httpMethod" equals "GET"
+  And the event "request.clientIp" is not null
```

Comment:
to verify that request context is sent in these scenarios (this is existing behaviour)